readme.md

```diff
@@ -30,10 +30,12 @@ The community is encouraged to port Rosé Pine to their favourite apps. To featu
 - [Terminal.app](https://github.com/rose-pine/terminal.app)
 - [Terminator](https://github.com/rose-pine/terminator)
 - [Termite](https://github.com/rose-pine/termite)
+- [Windows terminal](https://github.com/rose-pine/windows-terminal)
```

Comment:
Can you capitalize the T in Windows **T**erminal?

Reply:
Just did! Going to do the same on the repo, oops